### ISSUE TITLE
Error on load savefile

### ISSUE THREAD

**Issue by dairen** [raiguard/Factorio-LtnManager#45]
**Describe the bug**
just add a mod beatfinder
on screen, i see this message (cf below)
[version018_x75.zip](https://github.com/raiguard/Factorio-LtnManager/files/4493485/version018_x75.zip)

**Save file**
Upload the save file where the bug occured. If the file is too large for GitHub, use a file service such as Google Drive or Dropbox. If these aren't available, PM me (Raiguard) on the Factorio forums, or on Discord (Raiguard#7402).

If you can consistently reproduce the error from a new save, please upload the original save anyways, just in case.

**To Reproduce**
add beast finder

if i remove this mod, no crash

![image](https://user-images.githubusercontent.com/3027708/79579224-9b441900-80c7-11ea-9c7c-ef2e1afe069d.png)

**Comment by Tukamok** [comment]
I get the same thing.

Link to save and loaded mods:  [Factorio-ErrorReport-20200419.zip](https://drive.google.com/open?id=14dyyxb_nmniwKGH2SFKd55mgZTgOFHZy)

If you need any more details, let me know.

**Comment by raiguard** [comment]
I'm looking into this right now, and I can reproduce it. Now I just need to figure out why the flying frick it's happening...

**Comment by Tukamok** [comment]
Just for grins, I pulled the latest from the repo and get the same.

My guess is it's in cahoots with the girlfriend who thinks I play too much Factorio.  :)

**Comment by raiguard** [comment]
I figured it out... here's the long explanation:

I wrote a script that will gather and translate all of the materials in the game (items and fluids) into whatever language you have set, to allow you to search the inventory screen in your native language. This works by compiling the strings to be translated into a long array, and iterating it. To save on performance, it spreads this iteration out over multiple ticks (which is why the LTN Manager button doesn't become available right away).

The module will delete the player's data table once all translations are finished. Turns out, that if the iteration function happens to stop on the _exact last_ entry in the array, the module will finish receiving all of the translations, and delete the player table, without deregistering the iteration function. Then it tries to look at data that no longer exists, and crashes.

TL;DR: Black magic shenanigans, fixed for the next version. 🗡️ 

The update will actually be for RaiLuaLib, not LTN Manager.

**Comment by raiguard** [comment]
The RaiLuaLib update has been released, please try it now!

**Comment by Tukamok** [comment]
Looks good.  Thanks!
